_projects/gitmate_testing.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - GitMate
-desc: "GitMate.io is now an MVP with proper unit tests. However, we still lack support for quick integration tests on feature additions and bug fixes. Kudos, if you could help us with that."
+desc: "GitMate.io is now an MVP with proper unit tests. However, we still lack support for quick integration tests on
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.